src/main/frontend/extensions/srs.cljs

```diff
@@ -253,21 +253,22 @@
                        :or {use-cache? true}}]
    (when (string? query-string)
      (let [query-string (template/resolve-dynamic-template! query-string)
-           {:keys [query sort-by]} (query-dsl/parse repo query-string)
+           {:keys [query sort-by rules]} (query-dsl/parse query-string)
```

Comment:
I would like to have tested this fix but `dsl-query/query` still needs a little refactoring before srs can cleanly use it

Reply:
@tiensonqin Do you know of any example queries that would result in full-text-query?  being true or the result being a string? - https://github.com/logseq/logseq/blob/016efe8affacd4e3907cf142809b19831426d442/src/main/frontend/db/query_dsl.cljs#L577-L582 I haven't found any and I think I refactored all of build-query. Asking because my unfamiliarity there is what prevented me from this cleanup

Reply:
@logseq-cldwalker full-text query supports both {{query "text"}} and {{query "text"}}.